sql/core/src/main/scala/org/apache/spark/sql/SQLContext.scala

```diff
@@ -328,18 +328,19 @@ class SQLContext(@transient val sparkContext: SparkContext)
     def numPartitions = self.numShufflePartitions
 
     def strategies: Seq[Strategy] =
-      extraStrategies ++ (
```

Comment:
I think it's ok, it can at least save some space since `::` is 2 chars, `,` is 1 char.